cmsis-pack/lv_demos.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file lv_demos.h
```

Comment:
Would it be possible to copy this file by `gen_pack.sh`?

Reply:
No, this one is a patch for the original one.

Reply:
The original lv_demos.h include header files from other demo, especially from the music which is very big. So I have made this patch.

Reply:
Solved in the latest push https://github.com/lvgl/lvgl/pull/3021/commits/1703d5fc8569525b32ac4e707632ff5d72b0685e